Dockerfile

```diff
@@ -6,6 +6,9 @@ WORKDIR /opt/keycloak
 # Copy the realm file into the appropriate directory
 COPY tdei-realm.json /opt/keycloak/data/import/tdei-realm.json
 
+#ENV used by build script to enable health endpoint 
+ENV KC_HEALTH_ENABLED=true
```

Comment:
this is already applied on app service environment variables

Reply:
Build arguments and run arguments are different. We need this as part of build argument for packaging